### PR TITLE
videojs-standard Tracks

### PR DESCRIPTION
**Pull request by misteroneill**
## Description
Makes `src/js/tracks/` pass videojs-standard.

Previous discussion in #3461 

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/js/tracks/text-track-display.js [hunk 1]
```diff
@@ -2,29 +2,61 @@
  * @file text-track-display.js
  */
 import Component from '../component';
-import Menu from '../menu/menu.js';
```
Comment (gkatsev) [inline src/js/tracks/text-track-display.js]: These are no longer used in this file I assume?

File: src/js/tracks/text-track.js [hunk 2]
```diff
@@ -159,6 +160,8 @@ class TextTrack extends Track {
     let activeCues = new TextTrackCueList(tt.activeCues_);
     let changed = false;
     let timeupdateHandler = Fn.bind(tt, function() {
+      // This could be misleading. There is a getter on this property and if
```
Comment (gkatsev) [inline src/js/tracks/text-track.js]: Maybe better to say something like:
> Calling `this.activeCues` for the side-effects of updating itself due to it's nature as a getter function.
Do not remove or cues will stop updating.

Or something along the lines that explains better why it's being used here.

**Comment by gkatsev** [comment]
A potential enhancement to a comment.
LGTM, otherwise.

File: src/js/tracks/text-track.js (hunk 2 above)
Comment (misteroneill) [inline src/js/tracks/text-track.js]: That's better, I agree!

**Pull request merged**